ci/tasks/scripts/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -v -e -u -o pipefail
+
+source deploy-tools/aws-helpers.sh
+
+function deploy() {
+  local cluster_name service_name deploy_stage
+
+  deploy_stage="$(stage_name)"
+  cluster_name="${deploy_stage}-${CLUSTER_NAME}"
+  service_name="${SERVICE_NAME}-${deploy_stage}"
+
+  ecs_deploy \
+    "${cluster_name}" \
+    "${service_name}"
+}
```

Comment:
Is this behaviour different from Jenkins?
Don't remember actually going live previously, we still need to remap static IPs no?

Reply:
You are correct, the frontend shouldn't be deployed here, and should instead rely on the safe-restarter